GLTFSDK/Source/ExtensionsKHR.cpp

```diff
@@ -136,6 +136,7 @@ ExtensionSerializer KHR::GetKHRExtensionSerializer()
     extensionSerializer.AddHandler<Unlit, Material>(UNLIT_NAME, SerializeUnlit);
     extensionSerializer.AddHandler<DracoMeshCompression, MeshPrimitive>(DRACOMESHCOMPRESSION_NAME, SerializeDracoMeshCompression);
     extensionSerializer.AddHandler<TextureTransform, TextureInfo>(TEXTURETRANSFORM_NAME, SerializeTextureTransform);
+    extensionSerializer.AddHandler<TextureTransform, Material::NormalTextureInfo>(TEXTURETRANSFORM_NAME, SerializeTextureTransform);
```

Comment:
I think the inheritance of TextureInfo might be the issue here. Does it work for Occlusion right now? Also we need tests for this since it was a bug.

Reply:
Yeah, because it's inherited, Detail::MakeNameKey creates a unique name for NormalTextureInfo and so HasExtension can't find the extension for it.

Reply:
I've added support for `occlusionTexture` and added a couple of tests.